plugins/meta/iplink/README.md

```diff
@@ -0,0 +1,54 @@
+# iplink plugin
+
+## Overview
+
+This plugin changes some interface attributes, like 'ip link' command.
```

Comment:
This plugin provides a method to change interface attributes, linke `ip link` command.

Reply:
Fixed